ql/src/java/org/apache/hadoop/hive/ql/io/AcidUtils.java

```diff
@@ -2876,6 +2876,12 @@ private static boolean isLockableTable(Table t) {
     }
   }
 
+  public static boolean isNonNativeTable(Map<String, String> tblProps) {
```

Comment:
same function is already available: MetaStoreUtils.isNonNativeTable(), can't we use it?

Reply:
Yes correct. I will use it.

Reply:
could we create in MetaStoreUtils method that accepts Map<String, String> tblProps and reuse it in the original one?

Reply:
Done